libpod/container_internal_linux.go

```diff
@@ -2367,3 +2378,37 @@ func (c *Container) checkFileExistsInRootfs(file string) (bool, error) {
 	}
 	return true, nil
 }
+
+// getSecret gets a secret to mount into the container
+func (c *Container) getSecret(name string) (string, string, error) {
+	secretsDir := filepath.Join(c.runtime.store.GraphRoot(), "secrets")
+	manager, err := secrets.NewManager(secretsDir)
+	if err != nil {
+		return "", "", err
+	}
+	secr, data, err := manager.LookupSecretData(name)
+	if err != nil {
+		return "", "", err
+	}
+
+	secretFile := filepath.Join(c.config.StaticDir, secr.Name)
```

Comment:
This seems dangerous and subject to name conflicts (e.g., a secret named "containers").  I think we need to create these files in a subdir of `StaticDir`.

Reply:
Concur, we need a subdir.